psql2mysql.py

```diff
@@ -61,6 +61,37 @@ def _exclude_deleted(self, table, query):
             return query.where(table.c.deleted == 'False')
         return query.where(table.c.deleted == False)
 
+    def getTextColumns(self, table):
+        columns = table.columns
+        return [ c.name for c in columns if str(c.type) == 'TEXT' ]
+
+    def scanTableForLongTexts(self, table):
+        textColumns = self.getTextColumns(table)
+        if not textColumns:
+          return []
```

Comment:
no need of this as later you iterate over text_columns? eventually your logic will return `[]`, or fails for line 77?